index.html

```diff
@@ -170,15 +170,23 @@ <h2>Expected reorg <strong id="next-reorg-days"></strong></h2>
     should.equal(+parseDate('(since July 13, 2017)'), 1499896800000 /* big party */);
 });
 
+var NOW = +new Date();
+var ONE_DAY = 24 * 60 * 60 * 1000;
+
 var lis = $('li');
 var reorgs = [];
 for (var i = 0; i < lis.length; i++) {
     reorgs.push(parseDate(lis[i].textContent));
+    // Only use recent reorgs from the last 2 years if there were 10 reorgs
+    // or use the last 10 reorgs (exceeding the past 2 years).
```

Comment:
Maybe there could be an all-time average, a recent average, and we could use the recent one for the "yes/no" answer, while showing the all-time average below?

Reply:
The all-time is ~38 days, the recent one ~60. The first one felt too low based one recent experiences so I investigated why it had the value. The average for recent reorgs could become smaller than the all-time average and trigger the 'No' to the question if there was a reorg before the expected reorg date.